README

```diff
@@ -61,7 +61,7 @@ Running from a Checkout
 
 If you want to run from a checkout, you can perform the following commands::
 
-Install Prerequisites
+Install Prerequisites (on 22+ replace  yum with yum-deprecated or  dnf)
```

Comment:
I wonder if the yum/dnf remark shouldn't be placed one line underneath to not be in the title line